nix/solidity-package.nix

```diff
@@ -15,16 +15,20 @@
          xs);
 in
   pkgs.lib.makeOverridable (
-    attrs @ { test ? true, deps ? [], solc ? pkgs.solc, ... }:
+    { doCheck ? true
+    , deps ? []
+    , solc ? pkgs.solc
+    , test-hevm ? pkgs.dapp2.test-hevm
+    , ...
+    } @ attrs:
       pkgs.stdenv.mkDerivation (rec {
-        buildInputs = [pkgs.dapp2.test-hevm solc];
+        inherit doCheck;
+        buildInputs = [test-hevm solc];
```

Comment:
Can you add space after the brackets please?

Reply:
Yes